packages/styles/scss/patterns/listsection/_listsection-item.scss

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@mixin listsection-item {
+    .#{$prefix}--listsection-item {
+      color: $text-01;
+      &__title {
+        @include type-style('expressive-heading-02');
+      }
+      &__content {
+        @include type-style('body-long-02');
```

Comment:
I wonder if we set this to the `__container` if we need to keep declaring `body-long-02`?

Reply:
list section title uses $expressive-heading-05 & $text-01
list section body uses $body-long-02 & $text-01
list section group title uses $expressive-heading-04 & $text-01
list section item title uses $expressive-heading-01 & &text-01 and so on..
So if we set @include type-style('body-long-02') at container level , we will need to override again for child elements.
It makes more sense to set only  $text-01 at container level.

Reply:
But aren't you already overriding the child elements like `__title`? Another way to cut down on output is like this...

```scss
&__container,
&__content {
  @include type-style('body-long-02');
}
```